publify_core/app/views/archives_sidebar/_content.html.erb

```diff
@@ -1,6 +1,6 @@
 <% unless sidebar.archives.blank? %>
-  <h3 class="sidebar_title"><%= sidebar.title %></h3>
-  <div class="sidebar_body">
+  <h3 class="sidebar-title"><%= sidebar.title %></h3>
+  <div class="sidebar-body">
```

Comment:
These changes should not be in this PR. Always create a new branch off `master` for each new PR, so it only contains the fixes for the particular issue.